### PR TITLE
RadGridView-new-export-methods

### PR DESCRIPTION
**Pull request by StefanNenchev**

### CODE REVIEW

**Comment by StefanNenchev** [comment]
-Replaced the "Export Events" article as well ( located in the Events folder of RadGridView documentation)

**Comment by StefanNenchev** [comment]
Hi Petya,

Please review the Export Section that was refactored. Have in mind that the "Export" article within the "Events" section in RadGridView documentation was changed as well.

There are 2 new articles as well:

The "Modify content before exporting" article within the "How To" section is related to the new extension methods - ExportToWorkbook and ExportToRadFixedDocument. 

Added a how-to article regarding exporting numeric values when DataFormatString is applied for a certain column.
Thanks.

File: controls/radgridview/export/export.md [hunk 1]
```diff
@@ -36,11 +35,9 @@ It is the preferred method of exporting data. The method expects two parameters:
 * __UseSystemCultureSeparator__ - if set, the RadGridView will use the system List Separator string, specified in Control Panel's Regional Options, to separate cells. This property overrides the __ColumnDelimiter__ property. __Available in GridViewCsvExportOptions only__.
 
 
->RadGridView's Export() method will not export an image, but its path. You can only export more user friendly text, but not the image itself.
-
-The following example shows how to show a save file dialog asking the user to save the file in excel format:
+The following example shows how to display a "Save File" dialog asking the user to save the file in excel format:
```
Comment (PetyaSotirova) [inline controls/radgridview/export/export.md]: Example 1 shows...

File: controls/radgridview/export/export.md [hunk 2]
```diff
@@ -75,11 +72,8 @@ The following example shows how to show a save file dialog asking the user to sa
 	}
 {{endregion}}
 
-> As of __Q1 2015 RadGridView__ exposes two new methods: __ExportToXlsx__ and __ExportToPdf__, thus the  __Xlsx__ and __Pdf__ values from the __ExportFormat__ enum are obsolete.
-
-
 
-#### __VB.NET__
+#### __[VB.NET] Example2: Save RadGridView`s content in Excel file__
```
Comment (PetyaSotirova) [inline controls/radgridview/export/export.md]: Same snippet in different languages should be with the same number.

File: controls/radgridview/export/export.md [hunk 3]
```diff
@@ -106,11 +100,8 @@ The following example shows how to show a save file dialog asking the user to sa
 	 End If
 	End Sub
 {{endregion}}
-
->When opening an exported ".xls" file with Excel 2007 or later, you will get a warning message. The reason for it is that the RadGridView can export to HTML and ExcelML formats - in both cases not native for Excel 2007. In case of exporting to HTML format you will get such warning, however if you export to ExcelML format with XML extension you will not get it.
         
-
-In addition,__RadGridView__ provides a built-in methods to get the content of your grid view control in different formats:
+In addition,__RadGridView__ provides built-in methods to get the content of your GridView control in different formats:
```
Comment (PetyaSotirova) [inline controls/radgridview/export/export.md]: Missing interval prior RadGridView. Consider adding links to each article in the below list.

File: controls/radgridview/events/export-events.md [hunk 4]
```diff
@@ -48,45 +52,127 @@ You can subscribe to the events either declaratively or runtime like this:
...
-        
-As of version __Q1 2015__ we removed obsoleted class GridViewExportEventArgs and TextAlignment, VerticalAlignment, Background, Foreground, FontFamily, FontSize, FontWeight, Width, Height, Styles, Attributes properties from GridViewElementExportingEventArgs class. You can find detailed instructions on how to migrate your existing code related to styling in the [Backward Compatibility]({%slug radgridview-backward-compatibility%}#q1-2015) article.
+The event was introcuded in __Q1 2010 SP2__ as a direct replacement of the obsolete __Exporting__ event.
+The event takes argument of type __GridViewElementExportingEventArgs__ which expose the following properties:
 
-As you can see the event arguments' property __Element__ identifies the type of the currently exported element. The possible element types are defined in the __ExportElement__ enumeration:
+- __Cancel__: gets or sets a boolean value that indicates whether the event should be canceled or not.
```
Comment (PetyaSotirova) [inline controls/radgridview/events/export-events.md]: Upper-case in the explanation.

File: controls/radgridview/events/export-events.md [hunk 5]
```diff
@@ -48,45 +52,127 @@ You can subscribe to the events either declaratively or runtime like this:
...
+- __Context__: Gets the current context
+- __Element__: Gets the current element that is about to be exported
+- __Format__: Gets the current export format.
+- __Value__: 	Gets or sets the value to be exported.
+- __ShouldEncodeValue__: Gets or sets a value indicating whether special characters in the cell value will be escaped.
+- __VisualParameters__: Gets the visual export parameters. (Introduced in __Q3 2013__)
```
Comment (PetyaSotirova) [inline controls/radgridview/events/export-events.md]: Consider elaborating on what are those and how are they used.

File: controls/radgridview/export/export-formats/export-xlsx.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+* Telerik.Windows.Documents.Core.dll
+* Telerik.Windows.Documents.SpreadSheet.dll 
+* Telerik.Windows.Documents.SpreadSheet.FormatProviders.OpenXml.dll
+* Telerik.Windows.Zip.dll
+* Telerik.Windows.Controls.GridView.Export.dll
+
+>  __Telerik.Windows.Controls.GridView.Export.dll__ is a new binary introduced in __Q1 SP of 2015__. It delimits the exporting to __Xlsx__ functionality from __Telerik.Windows.Controls.GridView.dll__, so in order to use __ExportToXlsx__ method, the new dll should also be added.
+
+The method is overloaded:
```
Comment (PetyaSotirova) [inline controls/radgridview/export/export-formats/export-xlsx.md]: The method has two overloads

File: controls/radgridview/export/export-formats/export-xlsx.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+# ExportToXlsx
+
+The __ExportToXlsx__ method allows exporting to "Xlsx" format without the need to integrate RadGridView with *RadSpreadProcessing*. The method was introduced in __Q1 2015__.
+
+__Assembly References__
+
+ExportFormat.Xlsx method uses additional libraries so you need to add references to the following assemblies:
+
+* Telerik.Windows.Documents.Core.dll
+* Telerik.Windows.Documents.SpreadSheet.dll
```
Comment (PetyaSotirova) [inline controls/radgridview/export/export-formats/export-xlsx.md]: The casing should be "Spreadsheet"

File: controls/radgridview/export/how-to/export-header.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ description: Export Custom Header
 slug: gridview-export-troubleshooting-export-header
 tags: export,custom,header
 published: True
-position: 1
+position: 3
 ---
 
 # Export Custom Header
```
Comment (PetyaSotirova) [inline controls/radgridview/export/how-to/export-header.md]: Intro of the article is confusing - is the problem that headers are used or their export?
Consider making this an ordered list and include each snippet in its respective list item.

File: controls/radgridview/export/how-to/warning-message-excel.md [hunk 9]
```diff
@@ -26,6 +26,6 @@ The alert is a new security feature in Excel 2007 called Extension Hardening, wh
 
 __SOLUTION__
 
-You can [ export to ExcelML format]({%slug gridview-export%}) with XML extension. In this way you will not get such warning.
+You can [ export to ExcelML format]({%slug gridview-export%}) with XML extension. In such case you will avoid the warning.
```
Comment (PetyaSotirova) [inline controls/radgridview/export/how-to/warning-message-excel.md]: Latest versions of Excel are not very fond of the XML extension. Consider changing this to XLSX or at least add it as another option.

File: controls/radgridview/export/how-to/modify-exported-content.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+description: ExportFormat.Xlsx
+slug: gridview-export-xlsx
+tags: exportformat.xlsx
+published: True
+position: 7
+---
+
+# Modify RadGridView`s Content Before Exporting
+
+In __R1 2016__, we have introduced two new extension methods related to the exporting of RadGridView - *ExportToWorkBook()* and *ExportToRadFixedDocument()*. You can use them if you need to modify the content of the exported RadGridView and avoid styling the document manually.
```
Comment (PetyaSotirova) [inline controls/radgridview/export/how-to/modify-exported-content.md]: Casing should be "Workbook"

Comment (PetyaSotirova) [inline controls/radgridview/export/how-to/modify-exported-content.md]: Add info that those utilize SpreadProcessing and PdfProcessing and links to the libraries' documentation. That way customers will be able to find how to modify whatever they like.

**Pull request merged**